lib/flutter_opendroneid.dart

```diff
@@ -120,6 +139,35 @@ class FlutterOpenDroneId {
     await _api.setBtScanPriority(priority);
   }
 
+  /// Makes sure that all necessary permissions are granted, used before
+  /// performing any Bluetooth activity
+  static Future<bool> _ensureBluetoothPermissions() async {
```

Comment:
Tim, ze se to jmenuje `ensure` bych se asi nebal thrownout tu `PermissionMissingException` rovnou tady a budes mit jednodussi posilani tech chybejicich permission.

Nicmene necham to na tobe.